force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -83,7 +105,14 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Even" or "Odd", or null if the number is null.
      */
     public static String checkEvenOdd(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        if (a == null) {
+            return null;
+        } else if (Math.Mod(a, 2) == 0) {
+            return 'Even';
+        } else if (Math.Mod(a, 2) != 0) {
+            return 'Odd';
+        }
+        return null;
```

Comment:
One of these else if can be replace by an else because you're already returning null above